API/Controllers/QuestionsController.cs

```diff
@@ -34,6 +35,12 @@ await Context.Questions.Include(i => i.QuestionOptions).SingleOrDefaultAsync(i =
                 if (question == default(Question))
                     return ApiNotFound("Question does not exist!");
 
+                foreach (QuestionOption questionOption in question.QuestionOptions)
+                {
+                    if (string.IsNullOrEmpty(questionOption.PictureLink))
+                        questionOption.PictureLink = Environment.GetEnvironmentVariable("ASPNETCORE_IMGFALLBACKURL");
```

Comment:
Is it really required to put it into envar? Maybe we could just use constant instead?

Reply:
The URL will likely differ between development and production environments.

Reply:
Fair enough.